workbench-core/example/app/src/routes/datasetRoutes.ts

```diff
@@ -331,4 +331,32 @@ export function setUpDSRoutes(
       }
     })
   );
+
+  // remove dataset access permission
+  router.delete(
+    '/datasets/:datasetId/permissions',
+    wrapAsync(async (req: Request, res: Response) => {
+      if (req.params.datasetId.match(uuidWithLowercasePrefixRegExp(dataSetPrefix)) === null) {
+        throw Boom.badRequest('datasetid request parameter is invalid');
+      }
+      const validatedRequest = validateAndParse<AddRemoveAccessPermissionRequest>(
+        AddRemoveAccessPermissionParser,
+        req.body
+      );
+      let response: PermissionsResponse;
```

Comment:
do we need this `let` here?

Reply:
nope.

Reply:
was this fixed?

Reply:
Yes. Don't know why this isn't showing as outdated.